### PR TITLE
Perform all updates in one request

### PR DESCRIPTION
**Pull request by JosephAbbey**

### CODE REVIEW

**Comment by philipabbey** [comment]
I think we need to amend the default polling delay now that we perform a single large status update. If we don't force a change, we'll have rapid fire API calls which will wear down device batteries.

1. Can we set a new default in the properties?
2. Would 5 seconds be reasonable default delay?
3. Can we force the new default on devices when they install the update in order to prevent nasty surprises?
   - Perhaps change the name of the Property field so that it looks like a new one?

**Comment by philipabbey** [comment]
`propoerties.xml`:

```
<property id="poll_delay_combined" type="number">5</property>
```

**Review by philipabbey — CHANGES_REQUESTED** [review]

File: source/HomeAssistantTapMenuItem.mc [hunk 1]
```diff
@@ -53,6 +53,13 @@ class HomeAssistantTapMenuItem extends WatchUi.IconMenuItem {
         mData                 = data;
     }
 
+    function buildTemplate() as Lang.String or Null {
+        return null;
+    }
```
[the remark this answers is not in the document]
Comment (JosephAbbey) [reply source/HomeAssistantTapMenuItem.mc:58]: The standard is that every menu item has a `buildTemplate` and `updateState` function (there would be an interface if monkey c had those).

**Comment by JosephAbbey** [comment]
> * Perhaps change the name of the Property field so that it looks like a new one?

We need to write a new name and run translation

**Comment by philipabbey** [comment]
`settings.xml`:

```
  <setting
    propertyKey="@Properties.poll_delay_combined"
    title="@Strings.SettingsPollDelay"
  >
    <settingConfig type="numeric" min="0" />
  </setting>
```

We might need to insist on a minimum here.

**Comment by philipabbey** [comment]
On testing we decided:

- 5s default polling delay
- 0s minimum polling delay

No further code chages.

**Review by philipabbey — APPROVED** [review]

**Pull request merged**